101-jenkins-cluster-2-linux-1-win/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
remove the parameters for which you have defaultValues in the template... use proper GEN values for the rest, see: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#azure-devops-ci

Reply:
fixed